naming.md

```diff
@@ -0,0 +1,42 @@
+@devrel_team - some follow-up on our Service Naming Convention conversation we had a few weeks ago. Feedback welcome.
```

Comment:
Does this file really belong in the public repo?  I think very no.  Just the final outcome, in abbridged form, goes in CONTRIBUTING.md.  I wouldn't even have this in the history, so please rebase.

Reply:
This information is important, but need to go into `CONTRIBUTING.md`